<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LegendPanel</class>
 <widget class="QDockWidget" name="LegendPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>369</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>207</width>
    <height>369</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>235</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Legend Options</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="displayLegend">
      <property name="title">
       <string>Display Legend</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_4" rowstretch="0,0,0,0,0,0,0,0,0">
       <item row="7" column="0" colspan="5">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Division</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <widget class="QSpinBox" name="divInterval">
            <property name="suffix">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Thickness:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="divColor_btn">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Intervals:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="divThickness">
            <property name="suffix">
             <string>px</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="8" column="0" colspan="5">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Frameline</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="1">
           <widget class="QPushButton" name="frameColor_btn">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="frameThickness">
            <property name="suffix">
             <string>px</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Thickness:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="3" colspan="2">
        <widget class="QComboBox" name="layoutDirection">
         <item>
          <property name="text">
           <string>Vertical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Horizontal</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="setButton">
         <property name="text">
          <string>Set</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Layout Direction:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QComboBox" name="colormapFunction"/>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>ColorFunction:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="4">
        <widget class="QLineEdit" name="caption"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Caption:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
